app/models/navigation_link.rb

```diff
@@ -4,12 +4,16 @@ class NavigationLink < ApplicationRecord
   before_validation :allow_relative_url, if: :url?
   before_save :strip_local_hostname, if: :url?
 
+  enum section: { default: 0, other: 1 }, _suffix: :section
+
   validates :name, :url, :icon, presence: true
   validates :url, url: { schemes: %w[https http] }, uniqueness: { scope: :name }
   validates :icon, format: SVG_REGEXP
   validates :display_only_when_signed_in, inclusion: { in: [true, false] }
 
   scope :ordered, -> { order(position: :asc, name: :asc) }
+  scope :default_links, -> { where(section: 0) }
+  scope :other_links, -> { where(section: 1) }
```

Comment:
`enum` automatically adds scopes based on the options, they are called `default_section` and `other_section` in this case. I think we can stick with those instead of manually adding new ones.

```suggestion
```